.github/workflows/continuous-delivery.yml

```diff
@@ -336,14 +337,21 @@ jobs:
           # use git describe to get the nearest tag and use that to build the version (e.g. 1.4.0+dev24 or 1.4.0)
           commit_version=$(git describe --tags --match 'v*' "${commit_sha}"| sed -e 's/^v//; s/-g[0-9a-f]\+$//; s/-\([0-9]\+\)$/+dev\1/')
           commit_short=$(git rev-parse --short "${commit_sha}")
-          git_version_tag=$(git describe --tags --match 'v*')
+          
+          # extract branch name
+          if [ ${{ github.event_name }} != 'pull_request' ]
```

Comment:
should we add pull_request_target here, for pull_request_target we can get branch name from github_head_ref too